base-notebook/start.sh

```diff
@@ -40,7 +40,12 @@ if [ $(id -u) == 0 ] ; then
     # Ex: default NFS/EFS (no auto-uid/gid)
     if [[ "$CHOWN_HOME" == "1" || "$CHOWN_HOME" == 'yes' ]]; then
         echo "Changing ownership of /home/$NB_USER to $NB_UID:$NB_GID"
-        chown -R $NB_UID:$NB_GID /home/$NB_USER
+        chown $NB_UID:$NB_GID /home/$NB_USER
+    fi
+    if [ ! -z "$CHOWN_EXTRA" ]; then
+        for extra_dir in $(echo $CHOWN_EXTRA | tr ',' ' '); do
+            chown $NB_UID:$NB_GID $extra_dir
```

Comment:
Should `$CHOWN_EXTRA` be recursive? I know that risks running into the same problem, but if the user has a deeply nested set of directories that all need modification, listing them all is going to be a bear. Or should we expect users in that situation to modify the uid/gid ahead of the container launch, assuming they know what those values will be? 

@ewjoachim any thoughts on if this will satisfy your original use case?

Reply:
My original use case is: 
- I want to run docker on an existing `ipynb` in the current dir. I want to be able to edit the `ipynb` from both inside and outside the docker.

With your solution, what is the command I have to run ?

Reply:
Is it a single ipynb or do you expect to have a bunch of them in that directory?  I am considering adding a CHOWN_EXTRA_RECURSIVE setting (with a warning that it should only be done on known small directories)

Reply:
I added CHOWN_HOME_OPTS and CHOWN_EXTRA_OPTS just in case someone decides to add flags like -R to either chown.  It worries me for my use case, but that should not stop other people from having the opportunity